Fast-follows for merge jobs (#5308)
dbt-labs/docs.getdbt.com: website/docs/docs/deploy/merge-jobs.md

description: "Learn how to trigger a dbt job run when a Git pull request merges."
---

> # Merge jobs in dbt Cloud

Author: ```suggestion
```

You can set up a merge job to implement a continuous development (CD) workflow in dbt Cloud. The merge job triggers a dbt job to run when someone merges Git pull requests into production. This creates a seamless development experience where changes made in code will automatically update production data. 

By using CD in dbt Cloud, you can take advantage of deferral to build only the edited model and any downstream changes. With merge jobs, state will be updated almost instantly, always giving the most up-to-date state information in [dbt Explorer](/docs/collaborate/explore-projects).

## Prerequisites
- You have a dbt Cloud account. 
- You have set up a [connection with your Git provider](/docs/cloud/git/git-configuration-in-dbt-cloud). This integration lets dbt Cloud run jobs on your behalf for job triggering.

>    - If you're using a native [GitLab](/docs/cloud/git/connect-gitlab) integration, you need a paid or self-hosted account that includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). If you're using GitLab Free, merge requests will trigger CI jobs but CI job status updates (success or failure of the job) will not be reported back to GitLab.

Reviewer: active voice sugg
```suggestion
   - If you're using a native [GitLab](/docs/cloud/git/connect-gitlab) integration, you need a paid or self-hosted account that includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). If you're using GitLab Free, merge requests will trigger CI jobs but GitLab won't report back the CI job status updates (success or failure of the job).
```

Author: like your sugg but it changes the meaning of the sentence (updates not reported to GitLab vs GitLab not reporting updates)

## Prerequisites
- You have a dbt Cloud account. 
- You have access to the beta release for this functionality. 

> - You have set up a [connection with your Git provider](/docs/cloud/git/git-configuration-in-dbt-cloud). This integration lets dbt Cloud run jobs on your behalf for job triggering.

Author: double-checking: any other prereqs for Merge jobs that we should add?

Reviewer 2: Maybe the stuff around ensuring your git provider is set to send push notifications and link to the other documentation? Although that might be overkill for here.
